src/Stratis.Bitcoin.IntegrationTests/MinerTests.cs

```diff
@@ -709,36 +709,6 @@ public void GetProofOfWorkRewardForMinedBlocksTest()
             }
         }
 
-        [Fact]
-        public void Miner_Create_Block_Whilst_Connected_Syncs()
```

Comment:
We do connect, mine and sync literally on every integration test, so these are redundant.